ports/windows/mpconfigport.h

```diff
@@ -121,6 +121,7 @@
 #define MICROPY_PY_URANDOM          (1)
 #define MICROPY_PY_MACHINE          (1)
 #define MICROPY_PY_MACHINE_PULSE    (1)
+#define MICROPY_PY_THREAD           (1)
```

Comment:
Enabling threading definitely does impact performance, due to the GIL mutex's.  So this should be guarded by `#ifndef MICROPY_PY_THREAD` so that variants can disable it if needed.